Sources/TSCBasic/FileSystem.swift

```diff
@@ -224,41 +234,78 @@ public extension FileSystem {
     func getFileInfo(_ path: AbsolutePath) throws -> FileInfo {
```

Comment:
Should the default implementations above assert that the path is resolved?

Reply:
See the other comment about types and assertions.